mrtt-ui/src/language.js

```diff
@@ -3,11 +3,11 @@ const form = {
   checkboxGroupOtherLabel: 'Other',
   checkboxGroupOtherInputPlaceholder: 'If other, please state.'
 }
-const restorationAimsForm = {
+const multiselectWithOtherFormQuestion = {
   validation: {
     selectAtleastOneAim: 'Please select at least one aim.',
     clairfyOther: 'Please clarify other aim.'
```

Comment:
Im assuming you changed these prop names and values to 'item' in the other pr (not  enough time to check :) )

Reply:
yup updated!